docs/en/docs/encoders.md

```diff
@@ -0,0 +1,202 @@
+# Encoders
+
+Encoders are basically converters between datastructures and json. Its main use is in response were different return values are automatically
+parsed into json objects.
+
+For understanding what they are doing, it is helpful to see the manual usage
```

Comment:
To understand what they are doing, it is important and helpful to go through the [manual usage](#manual-usage)